components/TaskList/TaskList.test.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { render, screen, within } from "@testing-library/react";
+import { Checkbox, ListItem, UnorderedList } from "@chakra-ui/react";
+
+const fruits = [
+  { id: 1, name: "Bananas" },
+  { id: 2, name: "Apples" },
+  { id: 3, name: "Strawberries" },
+  { id: 4, name: "Grapes" },
+  { id: 5, name: "Oranges" },
+];
+function FruitList() {
```

Comment:
But if we don't import TasksList directly in order to test it, we aren't really testing the compoment, aren't we?

Reply:
Exactly but how can I render a component when its data is coming from DB?

Reply:
ah, good question. Can you import TaskList but still feed it with data "manually"? Or testing input and task List together?